RecoPixelVertexing/PixelTriplets/plugins/BuildFile.xml

```diff
@@ -12,7 +12,6 @@
 <use name="RecoPixelVertexing/PixelTriplets"/>
 <use name="RecoTracker/TkSeedingLayers"/>
 <use name="RecoTracker/TkTrackingRegions"/>
-<flags CXXFLAGS="-fno-math-errno"/>
```

Comment:
Is this an issue for the legacy code in this package ?

Reply:
this flag is default since years...